DOCS.md

```diff
@@ -1372,7 +1428,7 @@ __Arguments__
 ---------------------------------------
 
 <a name="sendMessage"></a>
-### api.sendMessage(message, threadID[, callback])
+### api.sendMessage(message, threadID[, callback[, messageID]])
```

Comment:
What does it mean for the [[]] to be nested? That you can't pass messageId without callback? Why?